programmers/난이도별/level01.하샤드 수/khw970421.js

```diff
@@ -0,0 +1,5 @@
+function solution(x) {
+    return (x%String(x).split('').reduce((a,b)=>Number(a)+Number(b))==0)?(true):false;
```

Comment:
```suggestion
    return (x % [...String(x)].reduce((a,b)=> a + Number(b), 0)) === 0;
```
이처럼 표현할 수 있습니다. 

` [...String(x)].reduce((a,b)=> a + Number(b), 0)` 이 부분은 따로 함수로 뺴어주면 가독성의 향상에 더 좋을 것 같습니다

Reply:
최대한 한줄에 처리하는것을 시도해보려고 저렇게 해보았는데 짧게하는 것도 좋지만 가독성 부분도 중요하군요